AspNetCoreServerSide/wwwroot/js/app.js

```diff
@@ -1,292 +0,0 @@
-                    {
-                        method: 'POST',
-                        cache: 'no-cache',
-                        body: new URLSearchParams(new FormData(document.querySelector('#frmCreate')))
-                    })
-                    .then((response) => {
-                        table.ajax.reload();
-                        $('#createModal').modal('hide');
-                        document.querySelector('#frmCreate').reset();
-                    })
-                    .catch((error) => {
-                        console.log(error);
-                    });
-            });
-
-        $(document)
-            .off('click', '.btnEdit')
-            .on('click', '.btnEdit', function () {
-                const id = $(this).attr('data-key');
-
-                fetch(`/Home/Edit/${id}`,
-                    {
-                        method: 'GET',
-                        cache: 'no-cache'
-                    })
-                    .then((response) => {
-                        return response.text();
-                    })
-                    .then((result) => {
-                        $('#editPartial').html(result);
-                        $('#editModal').modal('show');
-                    })
-                    .catch((error) => {
-                        console.log(error);
-                    });
-            });
-
-        $(document)
-            .off('click', '#btnUpdate')
-            .on('click', '#btnUpdate', function () {
-                fetch('/Home/Edit/',
-                    {
-                        method: 'PUT',
-                        cache: 'no-cache',
-                        body: new URLSearchParams(new FormData(document.querySelector('#frmEdit')))
-                    })
-                    .then((response) => {
-                        table.ajax.reload();
-                        $('#editModal').modal('hide');
-                        $('#editPartial').html('');
-                    })
-                    .catch((error) => {
-                        console.log(error);
-                    });
-            });
-
-        $(document)
-            .off('click', '.btnDelete')
-            .on('click', '.btnDelete', function () {
-                const id = $(this).attr('data-key');
-
-                if (confirm('Are you sure?')) {
-                    fetch(`/Home/Delete/${id}`,
-                        {
-                            method: 'DELETE',
-                            cache: 'no-cache'
-                        })
-                        .then((response) => {
-                            table.ajax.reload();
-                        })
-                        .catch((error) => {
-                            console.log(error);
-                        });
-                }
-            });
-
-        $('#btnExternalSearch').click(function () {
-            table.column('0:visible').search($('#txtExternalSearch').val()).draw();
-        });
-    }
-});
```

Comment:
Please don't delete the file. Replace the file and use the variables as I have suggested above.

Reply:
Done